jobs/ci-kubernetes-node-memcg-serial.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+# PROJECT="k8s-jkns-ci-node-e2e"
+test/e2e_node/jenkins/e2e-node-jenkins.sh test/e2e_node/jenkins/jenkins-memcg-serial.properties
```

Comment:
https://github.com/kubernetes/test-infra/blob/master/scenarios/kubernetes_kubelet.py#L119

that's the default already, you don't need this file.